### PR TITLE
[KYUUBI #2129] FlinkEngine throws UnsupportedOperationException in GetColumns

### PR DESCRIPTION
**Pull request by fsk119**
### _Why are the changes needed?_

Just throw the expected exception.

### _How was this patch tested?_
- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [x] [Run test](https://kyuubi.apache.org/docs/latest/develop_tools/testing.html#running-tests) locally before make a pull request

### CODE REVIEW

File: externals/kyuubi-flink-sql-engine/src/main/scala/org/apache/kyuubi/engine/flink/operation/FlinkSQLOperationManager.scala [hunk 1]
```diff
@@ -110,7 +110,14 @@ class FlinkSQLOperationManager extends OperationManager("FlinkSQLOperationManage
       catalogName: String,
       schemaName: String,
       tableName: String,
-      columnName: String): Operation = null
+      columnName: String): Operation = {
+    throw new UnsupportedOperationException(String.format(
```
Comment (SteNicholas) [inline externals/kyuubi-flink-sql-engine/src/main/scala/org/apache/kyuubi/engine/flink/operation/FlinkSQLOperationManager.scala:114]:
```suggestion
    throw new UnsupportedOperationException(s"
```

**Review by SteNicholas — APPROVED** [review]
LGTM. cc @yaooqinn

**Comment by yaooqinn** [comment]
it conflicts

**Review by yaooqinn — APPROVED** [review]

**Comment by SteNicholas** [comment]
@yaooqinn the failure of the CI isn't related to the changes of this pull request.

**Comment by yaooqinn** [comment]
thanks merged to master

**Pull request closed without merging**